diff --git a/src/linux/netlink/Netlink.cpp b/src/linux/netlink/Netlink.cpp
index f0677b7..47f5f8c 100644
--- a/src/linux/netlink/Netlink.cpp
+++ b/src/linux/netlink/Netlink.cpp
@@ -162,8 +162,10 @@ Netlink::DataPtr Netlink::dump_filter(int idx)
 		.msg_namelen = sizeof(nladdr),
 		.msg_iov = &iov,
 		.msg_iovlen = 1,
+		.__pad1 = 0,
 		.msg_control = NULL,
 		.msg_controllen = 0,
+		.__pad2 = 0,
 		.msg_flags = 0,
 	};
 	
